configure.ac

```diff
@@ -778,7 +774,7 @@ AC_HEADER_RESOLV
 ## The hton functions convert host endianness to network endianness.
 ## The ntoh functions convert network endianness to host endianness
 ## ======================================================================
-AC_CHECK_FUNCS([htonl htons ntohl ntohs __builtin_bswap16 __builtin_bswap32 __builtin_bswap64])
```

Comment:
as long as xdr doesn't use those defines

Reply:
The library does not use the bswap calls. It does use the htons, etc. so those are retained.